docs/ref/migration-operations.txt

```diff
@@ -228,8 +228,8 @@ The ``reverse_sql`` queries are executed when the migration is unapplied, so
 you can reverse the changes done in the forwards queries::
 
     migrations.RunSQL(
-        ["INSERT INTO musician (name) VALUES (%s);", ['Reinhardt']],
-        ["DELETE FROM musician where name=%s;", ['Reinhardt']],
+        [["INSERT INTO musician (name) VALUES (%s);", ['Reinhardt']]],
```

Comment:
The docs say, "You can also pass a list of strings or 2-tuples". Maybe the inner lists could be tuples or we could update that phrasing to avoid confusion. cc @MarkusH